distributionviewer/core/static/css/distributionviewer.styl

```diff
@@ -0,0 +1,48 @@
+$background-color = #fff;
+$normal-text = rgba(0,0,0,0.6);
+$mild-blue = #4A90E2;
+
+body {
+  background-color: $background-color;
+  color: $normal-text;
+  font-family: helvetica, sans-serif;
```

Comment:
Definitely. I've used django-pipeline for that in the past, but would also be up for using something more React-y if you have any recommendations.